src/createDispatcher.js

```diff
@@ -34,11 +34,17 @@ const kickstart = { type: '_INIT_' }
 export default function createDispatcher(opts = {}) {
   const dispatcher = new Subject()
 
+  // Question: what is this intended for? I see it used below in
+  // ways that make it seem like the Symbol is for scope hiding
+  // (as they were defined for), not sure what the intention is though
+  // Ah, this gets used to provide hidden annotations on a function the caller provided
   const identifier = Symbol()
   const cache = []
   const state = []
 
   let scheduler
+  // Now this is interesting, you're letting folks pick the scheduler
+  // Does this pick up derivative schedulers like the requestAnimationFrameScheduler
```

Comment:
Yea, it does ignore it. I phrased that not how I wanted to :D Actual question: Do you see any value in this taking any scheduler?